DepositUI/Pages/Deposit.razor

```diff
@@ -42,95 +38,55 @@
         </div>
     </div>
 }
-
-@code {
-    private string mode = "Main";
-
-    private List<DepositCalc> depositDetails;
-    private List<RequestError> requestErrors;
-
-    private string AmountStr { get { return amountStr; } set { amountStr = NumberInputCheck(value); } }
-    private string amountStr;
-
-    private string TermStr { get { return termStr; } set { termStr = NumberInputCheck(value); } }
-    private string termStr;
-
-    private string PercentStr { get { return percentStr; } set { percentStr = NumberInputCheck(value); } }
-    private string percentStr;
-
-    private bool getDepositError;
-    private HttpResponseMessage response = null;
-
-    private async Task Main() { }
-
-    private async Task History() { }
-
-    private async Task Details() { }
-
-    private async Task GetDepositCalc()
-    {
-        requestErrors = new List<RequestError>();
-        double amount;
-        double persent;
-        int term;
-
-        if (!Double.TryParse(NumberInputCheck(amountStr), out amount))
-        {
-            requestErrors.Add(new RequestError
-            {
-                PropertyName = "Amount",
-                ErrorMessage = "Amount is not number"
-            });
-            getDepositError = true;
-        }
-
-        if (!Double.TryParse(NumberInputCheck(percentStr), out persent))
-        {
-            requestErrors.Add(new RequestError
-            {
-                PropertyName = "Percent",
-                ErrorMessage = "Percent is not number"
-            });
-            getDepositError = true;
-        }
-
-        if (!Int32.TryParse(NumberInputCheck(termStr), out term))
-        {
-            requestErrors.Add(new RequestError
-            {
-                PropertyName = "Term",
-                ErrorMessage = "Term is not number"
-            });
-            getDepositError = true;
-        }
-
-        DepositModel deposit = new DepositModel()
-        {
-            Amount = amount,
-            Percent = persent,
-            Term = term
-        };
-
-        var request = new HttpRequestMessage(HttpMethod.Get, "https://localhost:44365/api/calculate");
-        request.Content = JsonContent.Create<DepositModel>(deposit);
-
-        var client = ClientFactory.CreateClient();
-        response = await client.SendAsync(request);
-        if (response.IsSuccessStatusCode)
-        {
-            depositDetails = await response.Content.ReadFromJsonAsync<List<DepositCalc>>();
-        }
-        else
+else if (mode == "History")
+{
+<div class="skrollable history">
+    <div class="row">
+        @for (int i = 0; i < 4; i++)
```

Comment:
What does the number mean? Introduce the number in a constant for better understanding